grafana-plugin/src/components/WorkingHours/WorkingHours.helpers.ts

```diff
@@ -82,3 +82,24 @@ export const getNonWorkingMoments = (startMoment, endMoment, workingHours) => {
 
   return nonWorkingMoments;
 };
+
+export const isInWorkingHours = (currentMoment: dayjs.Dayjs, workingHours, timezone) => {
+  const daysOfTheWeeks = {
+    1: 'monday',
+    2: 'tuesday',
+    3: 'wednesday',
+    4: 'thursday',
+    5: 'friday',
+    6: 'saturday',
+    7: 'sunday',
+  };
+  const currentDayOfTheWeek = daysOfTheWeeks[currentMoment.weekday()];
```

Comment:
There's no need for redefining week days, `dayjs` provides this out of the box with `dayjs().format('dddd')` therefore in your example I believe you can try

```jsx
const currentDayOfTheWeek = currentMoment.format('dddd').toLowerCase();
```

Haven't tested though, but I know `format('dddd')` gets you the long form of week day.